docker/conf/webauthn-fpm-pool.conf

```diff
@@ -0,0 +1,34 @@
+[global]
+daemonize = yes
+;Logs can be sent to stdout / err or to a file
+;error_log = /proc/self/fd/2
+error_log = /usr/local/var/log/php-fpm-error.log
+
+[www]
+;Logs can be sent to stdout / err or to a file
+;access.log = /proc/self/fd/2
+access.log = /usr/local/var/log/php-fpm-access.log
```

Comment:
Let's leave it out. I don't like files in containers, they tend to be deleted. 

Also, docker(-compose) logs provides a more standardized way to get your logs

Reply:
Hmmm, I added this specifically to make the `tail -f` command from the start script easier to read: it now follows both the apache and the php-fpm logs and it adds that nice header to tell you from which log (and daemon) the messages are coming from. 

Check it out, try to use it and see how you like it. If you still think it is a bad idea after that, we can remove it and find another way.